osu.Game/Screens/Select/FooterButton.cs

```diff
@@ -65,6 +66,9 @@ public FooterButton()
         {
             AutoSizeAxes = Axes.Both;
             Shear = SHEAR;
+            //Masks away one or two overlapping pixel
```

Comment:
We normally add a space after a comment prefix
```suggestion
            // Masks away one or two overlapping pixel
```